### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by gpetrou**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
 Dynamic Language Runtime
 ========================
-The Dynamic Language Runtime enables language developers to more easily create dynamic
-languages for the .NET platform. In addition to being a pluggable back-end for dynamic
-language compilers, the DLR provides language interop for dynamic operations on
-objects. The DLR has common hosting APIs for using dynamic languages as libraries or
-for scripting in your .NET applications.
+[![Windows Build Status](https://ci.appveyor.com/api/projects/status/53h9jt1bym8wunh1?svg=true)](https://ci.appveyor.com/project/AlexEarl/dlr)
```
Comment (gpetrou) [inline README.md]: I think I need something different than 53h9jt1bym8wunh1 here.

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+[![Linux/OSX Build Status](https://travis-ci.org/IronLanguages/dlr.svg?branch=master)](https://travis-ci.org/IronLanguages/dlr)
+[![NuGet](https://img.shields.io/nuget/v/DynamicLanguageRuntime.svg)](https://www.nuget.org/packages/MahApps.Metro/)
```
Comment (slide) [inline README.md]: Probably don't want to point to MahApps.Metro?

File: README.md (hunk 1 above)
Comment (slide) [inline README.md]: [Windows Build Status](https://ci.appveyor.com/project/AlexEarl/dlr)

**Comment by slide** [comment]
Stack exchange image is showing a broken image.

**Comment by gpetrou** [comment]
I did notice that but perhaps it is temporary? This was working before.
I see img.shields.io has the same issue.

**Comment by gpetrou** [comment]
I see this today, though I get the impression this works intermittently.
ItWorks

**Pull request merged**